lib/LedgerSMB/DBObject/Menu.pm

```diff
@@ -65,6 +66,22 @@ sub generate_section {
     return @{$self->{menu_items}};
 }
 
+=item generate_item()
```

Comment:
My thoughts were to generate the menu in a single request/response combination. If we use that approach, this function (and its associated SQL implementation below) are superfluous and probably better removed.